app/models/slide.rb

```diff
@@ -14,11 +15,19 @@ class Slide < ActiveRecord::Base
   belongs_to :user
   belongs_to :event
 
+  # validation macros
+  validates :title, :description, :user_id, :file, presence: true
+
+  # callbacks
+  # https://github.com/mperham/sidekiq/wiki/Problems-and-Troubleshooting#cannot-find-modelname-with-id12345
+  # after_commit :convert_file, :on => :create
+  after_create :convert_file
```

Comment:
try this patch https://gist.github.com/tf/c57e9072966712089cee
it's better then using `after_commit`, it's carrying too much details